inspector/src/components/graph/graphSupportingTypes.ts

```diff
@@ -1,3 +1,25 @@
-interface IDataset {
-    // empty for now, but will contain the shape of what a data set is.
+/**
+ * Defines what data is needed to graph a point on the graph.
+ *
+ * @interface IPoint
+ */
+interface IPoint {
+    x: number; // This will need to become a date of some kind with real data.

```

Comment:
typo in the comment

Reply:
I think I got it, I think it was just the wording and the interface tag (correct me if I'm wrong).